packages/tocco-ui/src/EditableValue/typeEditors/TimeEdit.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import StyledTimeEdit from './StyledTimeEdit'
+import Button from '../../Button'
+import {stringToDuration, numbersToTimeFormat} from '../utils'
+
+const TimeEdit = props => {
+  const hours = props.value ? props.value.hourOfDay : null
+  const minutes = props.value ? props.value.minuteOfHour : null
+
+  const timeString = numbersToTimeFormat(hours, minutes)
+
+  const handleChange = e => {
+    if (props.onChange) {
```

Comment:
is `onChange` a required prop? the prop types definition says so. in this case, the check here isn't necessary